.github/workflows/update-dependencies.yml

```diff
@@ -0,0 +1,20 @@
+name: Update dependencies
+
+on: [repository_dispatch]
+
+jobs:
+  update-webex-component-adapter-interfaces:
+    runs-on: ubuntu-latest
+    if: github.event.action == 'update-webex-component-adapter-interfaces'
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2-beta
+      - run: |
+          npm install @webex/component-adapter-interfaces --save
+          if [ -n "$(git status --porcelain)" ]; then
```

Comment:
[This is how we should commit back to the repo](https://github.com/actions/checkout#push-a-commit-using-the-built-in-token)